src/ui/view/governance/details/details.scss

```diff
@@ -173,6 +173,27 @@
   }
 }
 
+.okp4-dataverse-portal-governance-details-modal-top-element {
+  display: flex;
+  gap: 6px;
+  align-items: center;
+
+  h1 {
+    @include norse-font(700);
+    font-size: 24px;
+  }
+}
+
+.okp4-dataverse-portal-governance-details-modal-content {
+  @include noto-sans(400);
+  max-width: 293px;
+  overflow-wrap: break-word;
+}
+
+.okp4-dataverse-portal-governance-details-modal-button {
+  width: 120px !important;
```

Comment:
Instead of relying on `!important`, you can define a custom class to the modal main element, as it was done here:
https://github.com/okp4/dataverse-portal/pull/450/files#diff-a91ad3c3a02695048c65a6b719f332328e176ab4041a4d98320b826d998c3e0cR59

But it requires [changes](https://github.com/okp4/dataverse-portal/pull/450/files#diff-6995c4f4d1c510c4a2356cd2522ecc41619af4188854fb94abe3f216b27948f7) in Okp4Modal component, which has not been merged yet!

You can probably do the same, and fix eventual conflicts if you merge it after #450.

Reply:
Good point, fixed!